src/pages/docs/src/codegen/search.codegen

```diff
@@ -21,6 +21,10 @@ module.exports = function() {
           .shift()
           .substr(3),
       );
+      if(source === '' || source === undefined || source === null) {
```

Comment:
One of the three (apparently `undefined`) would have been sufficient. To cover all three you could have just written; 

```ts
if (source) {
  const content = readFileSync(source, 'utf8');
  return {
        ...meta,
        id: link,
        keywords: getTopKeywords(content),
        link,
        content,
  };
}

return {};
```

Reply:
eeehm ok :grimacing:
**adapted**